layouts/index.html

```diff
@@ -41,7 +41,7 @@ <h2>{{- .subtitle -}}</h2>
         </div>
         <div class="row middle-xs between-xs logos">
             {{- range .logos -}}
-                {{- partial (printf "icons/%s.html" .icon) -}}
+               <div class="col-xs-4 col-sm-2"> {{- partial (printf "icons/%s.html" .icon) -}} </div>
```

Comment:
Addid the `div` like with flexbox classes messes up the alignement of the logos, because the `div` is bigger than the logo. See the space on the left and on the right. You can have a look at what I did on https://www.inbeat.co/ home page to get inspired :)